core/src/main/java/org/mapfish/print/map/geotools/grid/RotationQuadrant.java

```diff
@@ -142,7 +167,7 @@ static RotationQuadrant getQuadrant(final double rotationDegrees) {
         if (rotation > 180 && rotation <= 270) {
             return QUADRANT_3;
         }
-        // CSOFF: MagicNumber
+        // CSON: MagicNumber
```

Comment:
The CSOFF line 160 is for turning off the warnings about "magic numbers" (instead of constants) from there to here. But the guy that did this, instead of turning them back ON, turned them off again. I've just fixed that.